core/deployment/src/main/java/io/quarkus/deployment/ConstructorPropertiesProcessor.java

```diff
@@ -0,0 +1,44 @@
+package io.quarkus.deployment;
+
+import java.beans.ConstructorProperties;
+
+import org.jboss.jandex.AnnotationInstance;
+import org.jboss.jandex.AnnotationTarget;
+import org.jboss.jandex.DotName;
+import org.jboss.jandex.IndexView;
+import org.jboss.jandex.MethodInfo;
+
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.CombinedIndexBuildItem;
+import io.quarkus.deployment.builditem.nativeimage.ReflectiveClassBuildItem;
+
+/**
+ * Registers all classes for reflection,
+ * that contain a constructor annotated with {@link ConstructorProperties}.
```

Comment:
Could you remove the link here? I would like to avoid having the import.

Otherwise it looks good to me.

Reply:
done, i missed that.